app/api/__init__.py

```diff
@@ -469,11 +469,11 @@
 
 # discount codes
 api.route(DiscountCodeListPost, 'discount_code_list_post', '/discount-codes')
-api.route(DiscountCodeList, 'discount_code_list', '/events/<int:event_id>/discount-codes',
+api.route(DiscountCodeList, 'discount_code_list', '/events/<int:event_id>/ticket-discount-codes',
           '/events/<event_identifier>/discount-codes', '/users/<int:user_id>/discount-codes',
           '/tickets/<int:ticket_id>/discount-codes')
 api.route(DiscountCodeDetail, 'discount_code_detail', '/discount-codes/<int:id>',
-          '/events/<int:event_id>/discount-code', '/event-invoices/<int:event_invoice_id>/discount-code',
+          '/events/<int:event_id>/event-discount-codes', '/event-invoices/<int:event_invoice_id>/discount-code',
```

Comment:
Why does DiscountCodeDetail have a URL of `/events/<int:event_id>/event-discount-codes`

Reply:
@iamareebjamal I changed the appropriate Url names which were specifically for `event-discount-codes` and `ticket-discount-codes`.

Reply:
But what will `/events/<int:event_id>/event-discount-codes` show. It is DiscountCodeDetail, not a list. Show me the response

Reply:
@iamareebjamal Ok, I am confused. See [this](https://api.eventyay.com/#discount-codes-ticket-discount-code-collection-get) and [this](https://api.eventyay.com/#discount-codes-get-discount-code-detail-of-an-event-get), we are using the same Url to retrieve a list and to retrieve Detail of a discount code.

Reply:
@iamareebjamal Yaa, it wouldn't give a list. It gives the detail of a discount code under an event, and not the discount code list which is of the type `event`. I will make appropriate changes. I got confused.